### DISCUSSION TITLE
Support for localization

### DISCUSSION THREAD

**Discussion by AzHicham** [mitsuhiko/minijinja#341] [Ideas]
Hello,
I would like to know if you are going too add support for locale translation like in [Jinja2 with gettext](https://svn.python.org/projects/external/Jinja-1.1/docs/build/i18n.html) 

thank you

**Comment by mitsuhiko** [comment]
I have no real plans on that. Mainly because Jinja2 uses gettext and I'm not convinced that this is the tool of choice these days.

**Reply by AzHicham** [reply]
I understand,
 I don’t know if fluent will become one day a standard, but the integration with Tera is pretty simple. https://github.com/XAMPPRocky/fluent-templates

**Reply by mitsuhiko** [reply]
For now I would recommend to try to build a third-party integration for minijinja. It's unlikely that I will have enough motivation to maintain this as part of the engine.
